CvGameCoreDLL/CvTeam.cpp

```diff
@@ -4118,6 +4118,16 @@ void CvTeam::setDefensivePact(TeamTypes eIndex, bool bNewValue)
 				{
 					kPlayer.updateTradeRoutes();
 				}
+				// Merijn: Berlaymont: 1 free specialist per defensive pact

```

Comment:
Please don't include effect implementations here.

Reply:
Oops. I didn't intend to upload this at all. I forgot to deselect it. I will reupload it and exclude it.

I coded some new wonders effects myself. The sole reason I did it is that it was fun to to. I didn't even expect you to use my code but rather want to code it yourself. I don't consider my time wasted as I enjoyed it, which is all matters to me.

Reply:
Done